EventKit/eventkit-cloud
# Handle 404 routes correctly

## Description of Improvement

This PR sets up the default 404 handler through django, and changes how traffic is routed through the httpd proxy so when running with DEBUG=False 404 routes will return a proper 404 status.

## How to test

Set DEBUG=False in the environment
Should be able to hit expected routes of the application (login, dashboard, create, etc) with expected redirect behaviors
Hitting a non-existent route should display default django 404 page, along with proper 404 status
Run export to verify normal functionality still working

## Quality Assurance 

The following items have been updated:
 
## Screenshots



:smile:
